js/analyzer.js

```diff
@@ -113,25 +117,53 @@ Analyzer.prototype.subheaderChecker = function() {
     return result;
 };
 
+/**
+ * check if the keyword contains stopwords
+ */
+Analyzer.prototype.stopwordChecker = function(){
+    matches = this.stringCounter(this.config.keyword, this.config.stopWords);
```

Comment:
Please extract `this.stringCounter(this.config.keyword, this.config.stopWords)` to something more verbose like `this.matchStopWords(this.config.keyword)`